Specs/Sequence/English/PhoneNumberModel.json

```diff
@@ -0,0 +1,958 @@
+{
+    "Input": "My phone number is 0176 12 34 56 78.",
+    "NotSupported": "javascript",
+    "NotSupportedByDesign": "python",
+    "Results": [
+        {
+        "Text": "0176 12 34 56 78",
+        "TypeName": "phonenumber",
+        "Resolution": {
+            "value": "0176 12 34 56 78"
+        }
+        }
+    ]
+}
+,
+{
+    "Input": "My phone number is 0176 123 456 78.",
+    "NotSupported": "javascript",
+    "NotSupportedByDesign": "python",
+    "Results": [
+        {
+        "Text": "0176 123 456 78",
+        "TypeName": "phonenumber",
+        "Resolution": {
+            "value": "0176 123 456 78"
+        }
+        }
+    ]
+}
+,
+{
+    "Input": "My phone number is 0176 1234 5678.",
+    "NotSupported": "javascript",
+    "NotSupportedByDesign": "python",
+    "Results": [
+        {
+        "Text": "0176 1234 5678",
+        "TypeName": "phonenumber",
+        "Resolution": {
+            "value": "0176 1234 5678"
+        }
+        }
+    ]
+}
+,
+{
+    "Input": "My phone number is 0176 12345678.",
+    "NotSupported": "javascript",
+    "NotSupportedByDesign": "python",
+    "Results": [
+        {
+        "Text": "0176 12345678",
+        "TypeName": "phonenumber",
+        "Resolution": {
+            "value": "0176 12345678"
+        }
+        }
+    ]
+}
+,
+{
+    "Input": "My phone number is 0176- 12345678.",
+    "NotSupported": "javascript",
+    "NotSupportedByDesign": "python",
+    "Results": [
+        {
+        "Text": "0176- 12345678",
+        "TypeName": "phonenumber",
+        "Resolution": {
+            "value": "0176- 12345678"
+        }
+        }
+    ]
+}
+,
+{
+    "Input": "My phone number is 0176-12345678.",
+    "NotSupported": "javascript",
+    "NotSupportedByDesign": "python",
+    "Results": [
+        {
+        "Text": "0176-12345678",
+        "TypeName": "phonenumber",
+        "Resolution": {
+            "value": "0176-12345678"
+        }
+        }
+    ]
+}
+,
+{
+    "Input": "My phone number is 0176/ 12345678.",
+    "NotSupported": "javascript",
+    "NotSupportedByDesign": "python",
+    "Results": [
+        {
+        "Text": "0176/ 12345678",
+        "TypeName": "phonenumber",
+        "Resolution": {
+            "value": "0176/ 12345678"
+        }
+        }
+    ]
+}
+,
+{
+    "Input": "My phone number is 0176/12345678.",
+    "NotSupported": "javascript",
+    "NotSupportedByDesign": "python",
+    "Results": [
+        {
+        "Text": "0176/12345678",
+        "TypeName": "phonenumber",
+        "Resolution": {
+            "value": "0176/12345678"
+        }
+        }
+    ]
+}
+,
+{
+    "Input": "My phone number is 017612345678.",
+    "NotSupported": "javascript",
+    "NotSupportedByDesign": "python",
+    "Results": [
+        {
+        "Text": "017612345678",
+        "TypeName": "phonenumber",
+        "Resolution": {
+            "value": "017612345678"
+        }
+        }
+    ]
+}
+,
+{
+    "Input": "My phone number is 622 15 31 23.",
```

Comment:
All Chile/Argentina numbers fall into the general regex?

Reply:
They do fall into it.